backend/src/helpers/UserHelper.js

```diff
@@ -0,0 +1,42 @@
+// Priorities are set from the lowest to the highest.
+const statePriorities = [ 'away', 'viewing', 'commenting', 'editing', 'merging' ];
+
+class UserHelper {
+	static getUserListFromSessions( issueSessions ) {
+		const users = {};
+		for ( const socketId in issueSessions ) {
+			if ( !users[ issueSessions[ socketId ].id ] ) {
+				users[ issueSessions[ socketId ].id ] = [ issueSessions[ socketId ] ];
```

Comment:
IMO just initialize the array here without any data and get rid of the `continue` instruction.

Reply:
Sure, it's simplier.